packages/react-client/src/ReactFlightClient.js

```diff
@@ -2752,32 +2752,25 @@ function createFakeFunction<T>(
     code = comment + code;
   }
 
-  if (filename.startsWith('/')) {
-    // If the filename starts with `/` we assume that it is a file system file
-    // rather than relative to the current host. Since on the server fully qualified
-    // stack traces use the file path.
-    // TODO: What does this look like on Windows?
-    filename = 'file://' + filename;
-  }
```

Comment:
The host should deal with this instead (e.g. in Node.js via `url.pathToFileURL`). For CommonJS it'd be the wrong thing to do since the sourcemaps of CJS files are keyed by their file path in Node.js not their file URL.